<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/">
  <channel>
    <title>linux on A Developer&#39;s Note</title>
    <link>https://shiroyuki.dev/en/tags/linux/</link>
    <description>Recent content in linux on A Developer&#39;s Note</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en-us</language>
    <copyright>Aghits Nidallah - shiroyuki.dev &amp;copy;2023</copyright>
    <lastBuildDate>Thu, 23 Feb 2023 12:00:20 +0700</lastBuildDate><atom:link href="https://shiroyuki.dev/en/tags/linux/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>How to fix Could Not Resolve Hostname github.com on Linux</title>
      <link>https://shiroyuki.dev/en/article/cara-mengatasi-could-not-resolve-hostname-github-com-pada-linux/</link>
      <pubDate>Thu, 23 Feb 2023 12:00:20 +0700</pubDate>
      
      <guid>https://shiroyuki.dev/en/article/cara-mengatasi-could-not-resolve-hostname-github-com-pada-linux/</guid>
      <description>This situation began when I planned to do a git pull on a project in CentOS 7 server. However, unlike my previous experience, the response that appeared was as follows:
1 2 3 ssh: Could not resolve hostname github.com: Temporary failure in name resolution fatal: Could not read from remote repository. Please make sure you have the correct access rights and the repository exists. It was very surprising, because previously I could always do the pull without any problems.</description>
    </item>
    
  </channel>
</rss>
